components/movementsensor/client_test.go

```diff
@@ -43,7 +43,7 @@ func TestClient(t *testing.T) {
 	heading := 202.
 	props := &movementsensor.Properties{LinearVelocitySupported: true}
 	acc := map[string]float32{"x": 1.1}
-	rs := []interface{}{loc, alt, r3.Vector{0, speed, 0}, r3.Vector{0, 0, ang}, heading, ori}
+	rs := []interface{}{loc, alt, r3.Vector{0, speed, 0}, r3.Vector{0, 0, ang}, r3.Vector{0, 0, 0}, heading, ori}
```

Comment:
I get nervous about adding things into the middle of a list, because anything that was indexing into it will need to be updated (e.g., if there's ever a line like `heading := rs[4]`, it's now broken). Why not add the new part onto the very end?